### ISSUE TITLE
Doesn't work for Ninja Gaiden Sigma

### ISSUE THREAD

**Issue by Chucky2401** [sharkusmanch/playnite-ludusavi-restic#10]
Hello,
I have an issue with a game in particularly.

For Ninja Gaiden Sigma, it doesn't work. First, the game name is exactly `Ninja Gaiden Σ`, howerver ludusavi will not find the game.

And, in the log I have only this:
```
22-01 20:58:14.173|DEBUG|LudusaviRestic#BaseCommand:D:\Utilisateurs\TheBlackWizard\Jeux\Backup\ludusavi.exe
22-01 20:58:14.173|DEBUG|LudusaviRestic#BaseCommand:backup --api --try-update --preview --merge "Ninja Gaiden Σ"
22-01 20:58:14.906|DEBUG|LudusaviRestic#BackupTask:Got Ninja Gaiden Σ data from ludusavi
```

It seems restic is never called. Maybe the Sigma character? But I can't change the game name...

Thank you for your help.

**Comment by Chucky2401** [comment]
I found the solution in your code.

In the `CommandResult.cs`, replace this line:
```c#
public string StdOut { get { return this.stdout; } }
```

by this:
```c#
public string StdOut { get {
                byte[] bytes = Encoding.Default.GetBytes(this.stdout);
                string myString = Encoding.UTF8.GetString(bytes);
                return myString;
            } }
```

to return it as UTF8 and it works!

```powershell
restic -r D:\Utilisateurs\TheBlackWizard\Jeux\Backup\Playnite\Archives snapshots -compact
enter password for repository:
repository 3076cc80 opened successfully, password is correct
ID        Time                 Host      Tags
-----------------------------------------------------------------
28a7062d  2022-01-21 14:06:57  Zoukzouk  Diablo III
9b237d89  2022-01-21 18:30:21  Zoukzouk  A Plague Tale: Innocence
3647104c  2022-01-22 19:32:14  Zoukzouk  Minecraft
4a3af328  2022-01-22 20:56:59  Zoukzouk  Minecraft
5572b3e9  2022-01-23 15:50:02  Zoukzouk  Diablo III
78afcf5a  2022-01-23 16:19:27  Zoukzouk  Ninja Gaiden Σ
-----------------------------------------------------------------
6 snapshots
```

```log
23-01 16:19:25.848|DEBUG|LudusaviRestic#BaseCommand:D:\Utilisateurs\TheBlackWizard\Jeux\Backup\ludusavi.exe
23-01 16:19:25.848|DEBUG|LudusaviRestic#BaseCommand:backup --api --try-update --preview --merge "Ninja Gaiden Σ"
23-01 16:19:27.817|DEBUG|LudusaviRestic#BackupTask:Got Ninja Gaiden Σ data from ludusavi
23-01 16:19:27.817|DEBUG|LudusaviRestic#BaseCommand:C:\Users\The Black Wizard\scoop\shims\restic.exe
23-01 16:19:27.817|DEBUG|LudusaviRestic#BaseCommand:backup --tag  "Ninja Gaiden Σ" --files-from-verbatim "C:\Users\The Black Wizard\AppData\Local\Temp\tmpA237.tmp"
```

**Comment by sharkusmanch** [comment]
Thanks for investigating!

Will take a look sometime this week. Feel free to open a PR if you want

**Comment by sharkusmanch** [comment]
Fixed in v0.2.2
